app/TermsOfUseVersion.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class TermsOfUseVersion extends Model {
+    use HasFactory;
+
+    protected $table = 'tou_versions';
+
+    const FIELDS = [
+        'version',
+        'active',
+        'acceptance_deadline',
+        'content',
+    ];
+
+    protected $fillable = self::FIELDS;
+
+    protected $visible = self::FIELDS;
+
+    protected $casts = [
+        'version' => 'string',
+        'active' => 'boolean',
+        'acceptance_deadline' => 'datetime',
+        'content' => 'string',
+    ];
+
+    public static function latestVersion(): ?self {
```

Comment:
change to activeVersion()

Reply:
Not sure how I feel about relying on us remembering to only set one active ToU version. In my head, I was thinking that `active` would mean "not draft; allowed to show to users" and we would always show the most recent version. How about naming this function `latestActiveVersion()` and adding an `ORDER BY version` clause to always ensure the latest version is returned?